install.sh

```diff
@@ -33,6 +33,11 @@ if [ -z "$DOWNLOAD_URL" ]; then
 	DOWNLOAD_URL=$DEFAULT_DOWNLOAD_URL
 fi
 
+DEFAULT_REPO_FILE="docker-ce.repo"
+if [ -z "$REPO_FILE" ]; then
+	REPO_FILE="$DEFAULT_REPO_FILE"
```

Comment:
Personally I would set REPO_FILE="docker-ce.repo" in the if statement, and not create an extra variable. 
If we end up needing to use the string "docker-ce.repo" multiple times (even if a REPO_FILE is defined beforehand) then we can create the extra variable.

Reply:
Historically we've set default values this way in this script, see lines 26-34